### PR TITLE
Fixes the broken cancer that is canned laughter spacedong please test your shit coders please codecheck properly thank you

### PR DESCRIPTION
**Pull request by Kierany9**
- Canned laughter now actually metabolizes at a rate of 0.2 units per tick instead of staying in your body forever.
- Laughing now has a 10% chance of happening per tick instead of happening every single tick.
- You now need to be able to talk to laugh.
- 'Nothing' now works as an antidote for canned laughter because mimes.

### CODE REVIEW

**Comment by Voratius** [comment]
yes pls +1/3rd

**Comment by calibraptor** [comment]
+1/3rd

Also LYNCH SPACEDONG

**Comment by calibraptor** [comment]
This is a fix this is a fix this is a fix merge it NOW

**Comment by Kierany9** [comment]
travis

**Comment by GeneralGoose** [comment]
btw kan't you use pick() for sounds instead of that laughnum variable?
and shouldn't laugh be in emotes.dm?

**Comment by JohnGinnane** [comment]
Goose is right, this bypasses a lot things to force people to laugh. Should we fix this now too?

**Comment by dreamcrab** [comment]
ja

**Comment by McBawbaggings** [comment]
Yeah okay +1

**Comment by Kierany9** [comment]
Good point, added a check to make sure you can actually talk before laughing

**Comment by GeneralGoose** [comment]
kool

**Comment by Kokojos** [comment]
Code is kosher

File: html/changelogs/Kierany9 - goddamnit spacedong [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Kierany9
```
Comment (Carbonhell) [inline html/changelogs/Kierany9 - goddamnit spacedong]: you FORGOT the DAMN extension .yml REEEE

File: code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm [hunk 2]
```diff
@@ -591,19 +591,21 @@
...
-		if(laughnum == 2)
-			playsound(get_turf(M), 'sound/voice/manlaugh2.ogg', 50, 1)
-	else if(M.gender == FEMALE)
-		playsound(get_turf(M), 'sound/voice/womanlaugh.ogg', 65, 1)
-	else//non-binary gender just sounds like a man
-		playsound(get_turf(M), 'sound/voice/manlaugh1.ogg', 50, 1)
+	if(holder.has_reagent("nothing"))
+		metabolization_rate = 5 * REAGENTS_METABOLISM
+	else
+		metabolization_rate = 0.5 * REAGENTS_METABOLISM
```
Comment (Carbonhell) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm]: what no why is this here

File: code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm [hunk 3]
```diff
@@ -591,19 +591,21 @@
...
+		else//non-binary gender just sounds like a man
 			playsound(get_turf(M), 'sound/voice/manlaugh1.ogg', 50, 1)
-		if(laughnum == 2)
-			playsound(get_turf(M), 'sound/voice/manlaugh2.ogg', 50, 1)
-	else if(M.gender == FEMALE)
-		playsound(get_turf(M), 'sound/voice/womanlaugh.ogg', 65, 1)
-	else//non-binary gender just sounds like a man
-		playsound(get_turf(M), 'sound/voice/manlaugh1.ogg', 50, 1)
+	if(holder.has_reagent("nothing"))
+		metabolization_rate = 5 * REAGENTS_METABOLISM
```
Comment (Carbonhell) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm]: nig that's gonna be *5ed for every on_mob_life check,that's probs not what you want,is it

Comment (McBawbaggings) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm]: Isn't that just 5 * 0.4 though? It's a hard coded define?

Comment (Carbonhell) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm]: nvm all good,i mixed the vars,same for down here

**Comment by Spacedong** [comment]
lynch me +1
